.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we want this file in .gitignore as well?

Reply:
We should pull both of these, good catch.